store/sql_pull_request_store.go

```diff
@@ -6,42 +6,40 @@ package store
 import (
 	"database/sql"
 	"fmt"
+	"strings"
 
 	"github.com/mattermost/mattermost-mattermod/model"
 )
 
 type SQLPullRequestStore struct {
 	*SQLStore
+	insertQuery string
+	updateQuery string
 }
 
 func NewSQLPullRequestStore(sqlStore *SQLStore) PullRequestStore {
-	s := &SQLPullRequestStore{sqlStore}
+	s := &SQLPullRequestStore{SQLStore: sqlStore}
+	var b strings.Builder
+	b.WriteString("INSERT INTO PullRequests ")
+	b.WriteString("(RepoOwner, RepoName, FullName, Number, Username, Ref, Sha, Labels, State, BuildStatus, ")
+	b.WriteString("BuildConclusion, BuildLink, URL, CreatedAt, Merged, MaintainerCanModify) ")
+	b.WriteString("VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16)")
+	s.insertQuery = b.String()
```

Comment:
@agnivade could yo tell me if this is the approach we are going to take? Any comment is welcome